src/components/Output.tsx

```diff
@@ -371,9 +371,7 @@ export const Output = ({
   }
 };
 
-const Styled: Record<string, AnyStyledComponent> = {};
-
-Styled.Output = styled.output<{ withParentheses?: boolean }>`
+const _OUTPUT_STYLES = styled.output<{ withParentheses?: boolean }>`
```

Comment:
nit: Curious why `_OUTPUT_STYLES` is split out

Reply:
split it out because other styles extend it - figured it was best to just abstract it out